README.md

```diff
@@ -2,7 +2,7 @@
 
 [![CircleCI][circleci-image]][circleci-url]
 
-The Salesforce Commerce SDK allows easy interaction with the Salesforce B2C Commerce platform APIs on the Node.js runtime.
+The Salesforce Commerce SDK allows easy interaction with the Salesforce B2C Commerce platform APIs on the Node.js runtime.  If you require a lightweight SDK that works in a browser and Node.js for shopper experiences please refer to our (Isomorphic SDK)[https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic]
```

Comment:
Suggest change from "If you require a lightweight..." to "For a more lightweight SDK, which works in a browser and Node.js for the shopper experience, see [Isomorphic..."

Reply:
Changed!